storage/libradb/src/schema/transaction/mod.rs

```diff
@@ -43,14 +36,13 @@ impl KeyCodec<TransactionSchema> for Version {
     }
 }
 
-impl ValueCodec<TransactionSchema> for SignedTransaction {
+impl ValueCodec<TransactionSchema> for Transaction {
     fn encode_value(&self) -> Result<Vec<u8>> {
-        let event: types::proto::types::SignedTransaction = self.clone().into();
```

Comment:
`event` :)

Reply:
not me.. not me.. 🤡 https://github.com/libra/libra/commit/ab4b9ad20bc458a153f35ecd4175804c1f1d6199